src/index.mjs

```diff
@@ -156,6 +156,7 @@ function _plot(data, options) {
         { style: "transform: translate(-15%, 55%)" }
       )}
       ${yPoints.map((p, i) => {
+        p = new Intl.NumberFormat('en-US').format(p);
```

Comment:
Can we please make `"en-US"` an option, e.g. 

```
{
  "...": "...",
  "locale": "en-US",
  "...": "...",
}
```

Reply:
definitely